### PR TITLE
Release 2.0.0

### PR DESCRIPTION
**Pull request by ashwini-mhatre**
##### ISSUE TYPE

- Docs Pull Request

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

File: changelogs/CHANGELOG.rst [hunk 1]
```diff
@@ -11,6 +11,8 @@ v2.0.0
 Major Changes
 -------------
 
+- Please refer to ansible.netcommon `changelog <https://github.com/ansible-collections/ansible.netcommon/blob/main/changelogs/CHANGELOG.rst#ansible-netcommon-collection-release-notes>`_ for more details.
+- Requires ansible.netcommon v2.0.0+ to support `ansible_network_single_user_mode` and `ansible_network_import_modules`.
```
Comment (NilashishC) [inline changelogs/CHANGELOG.rst:15]: I don't know why the changelog tool swaps the order for this fragment. But reversing it probably makes more sense for the user to understand why this is a major release.

Comment (ashwini-mhatre) [reply changelogs/CHANGELOG.rst:15]: okay

Comment (ashwini-mhatre) [reply changelogs/CHANGELOG.rst:13]: Ack

**Review by NilashishC — APPROVED** [review]

**Comment by ashwini-mhatre** [comment]
recheck

**Comment by ashwini-mhatre** [comment]
test

**Comment by ashwini-mhatre** [comment]
recheck

**Comment by ashwini-mhatre** [comment]
recheck

**Comment by ashwini-mhatre** [comment]
Recheck

**Comment by ashwini-mhatre** [comment]
Recheck

**Comment by justjais** [comment]
recheck

File: galaxy.yml [hunk 2]
```diff
@@ -2,7 +2,7 @@
 authors:
   - Ansible Network Community (ansible-network)
 dependencies:
-  "ansible.netcommon": "*"
+  "ansible.netcommon": "2.0.0"
```
Comment (justjais) [inline galaxy.yml:5]:
```suggestion
  "ansible.netcommon": ">=2.0.0"
```

**Review by justjais — CHANGES_REQUESTED** [review]

**Comment by justjais** [comment]
@ashwini-mhatre I don’t see docs generated for newly introduced Bgp modules, have u ran the collection prep doc script for generating the docs?

**Comment by ashwini-mhatre** [comment]
@justjais Actually https://github.com/ansible-collections/cisco.iosxr/pull/110 this one has those changes.I created 110 PR for this release. but then we have decide we need to update galaxy.yaml so for that change I raised PR 111 beacuse of this isssue the current PR does not have those changes related to docs those are already in main branch.

**Comment by ashwini-mhatre** [comment]
recheck

**Review by justjais — APPROVED** [review]

**Pull request merged**